.github/workflows/build-test-deploy.yml

```diff
@@ -114,31 +114,195 @@ jobs:
             nix-env -iA nixpkgs.cachix
           authToken: "${{ secrets.CACHIX_AUTH_TOKEN }}"
           name: composable-community
-
-      - name: Cargo fmt
-        run: |
+      - run: |
           cachix watch-exec -j 16 -c 0 composable-community nix -- build .#cargo-fmt-check --no-update-lock-file --show-trace -L
 
-      - name: Taplo CLI
-        run: |
+  taplo-cli-check:
+    name: "Taplo cli check"
+    needs: common-deps
```

Comment:
I think taplo does not need common deps?

Reply:
yes, you are right.

Reply:
fixed.